README.md

```diff
@@ -1,43 +1,153 @@
 
-## Global requirements
-1. authentication/authorization
-2. manage exercises (crud)
+Apply the TypeScript static typing system. try to use TypeScript for the majority of React components! installed typescript, add support for TypeScript linting, and use either .ts or .tsx extension for the files to apply TypeScript type-checking on.
 
-### Import requirements
-1. exercise (add exercise, [pushup, pull, number, completed/not] save to storage , loop)
-2. exercise log
+Use the `BLANK_README.md` to get started.
 
-## highlight important things for next time
-try to make it detailed
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+### Screenshots
 
-guide
-descript, feature, basic layout, functionality
+### Built With
 
-1. feature: add new, exercise, layout, functionality: add button, exercise type, reps, done/not
+A few of the frameworks/libraries that were used to build the project are:
 
-2.
-3.
-4.
+* [![React][React.js]][React-url]
+* Firebase
+* React Redux 
+* Axios
+* React Router
+* EsLint and AirBnb
+* Typescript
 
-Creating the workout scheme in React
-In our app, we want the user to be able to:
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-Add and remove exercises
-Add and remove sets
-For each set, add weight and reps
-For each set, mark as finished or unfinished
+### Wireframe using Figma 
+  ### Getting Started The App build out with simple wireframes using Figma.
+<img src="/assets/images/homewireframe.png"
+        alt="home page"
+             height="80">
+<img src="/assets/images/signinwireframe.png"
+        alt="sign in page"
+             height="80">
+<img src="/assets/images/signupwireframe.png"
+        alt="sign up page"
+             height="80">
+### How To Backup & Restore Your Workouts
+You can export individual workouts or create an export of all your workouts and import them later. You can also transfer them to another device.
 
+<!-- GETTING STARTED -->
+## Getting Started
 
-if first time signin
-take to profile
-auto profile with info
+This is an example of how you may give instructions on setting up your project locally.
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+* npm
+  ```sh
+  npm install npm@latest -g
+  ```
+
+### Installation
+### Installation
+_Below is an example of how you can instruct your audience on installing and setting up your app. This template doesn't rely on any external dependencies or services._
+### Installation
+1. Get a free account at for firebase to create your own collection and add it to your .env file to connect your API to your Data base
```

Comment:
This bullet is a little confusing... maybe combine it with bullet 4 or reference bullet 4 to clarify?